### ISSUE TITLE
Deploying to Digital Ocean as an app. Keras load_model not working

### ISSUE THREAD

**Issue by reconstructive** [mtobeiyf/keras-flask-deploy-webapp#37]
Has anybody come an issue deploying a custom model (h5 or saved_model format) and receiving the following when trying to load the model using tf.keras.models.load_model:

`radiographyml | 17:42:33 Traceback (most recent call last):
radiographyml | 17:42:33   File "app.py", line 53, in <module>
radiographyml | 17:42:33     model = load_model(MODEL_PATH)
radiographyml | 17:42:33   File "/usr/local/lib/python3.7/site-packages/tensorflow/python/keras/saving/save.py", line 186, in load_model
radiographyml | 17:42:33     loader_impl.parse_saved_model(filepath)
radiographyml | 17:42:33   File "/usr/local/lib/python3.7/site-packages/tensorflow/python/saved_model/loader_impl.py", line 113, in parse_saved_model
radiographyml | 17:42:33     constants.SAVED_MODEL_FILENAME_PB))
radiographyml | 17:42:33 OSError: SavedModel file does not exist at: ./models/my_model.h5/{saved_model.pbtxt|saved_model.pb}`

The model is quite big, and I'm using git lfs to upload it to github. The repo is synced to Digital Ocean and refreshes each time a new commit is made. I've been stuck for a while here, trying different models, versions of TF, python, filepaths etc etc

**Comment by reconstructive** [comment]
```
radiographyml | 09:50:05 OpenBLAS WARNING - could not determine the L2 cache size on this system, assuming 256k
radiographyml | 09:50:07 2021-02-20 07:50:07.502411: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
radiographyml | 09:50:07 2021-02-20 07:50:07.502865: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
radiographyml | 09:50:20 Model loaded. Check http://127.0.0.1:5000/
radiographyml | 09:50:20 path /app
radiographyml | 09:50:20 ['.git', '.gitattributes', 'Dockerfile', 'LICENSE', 'Procfile', 'README.md', '__pycache__', 'app.py', 'models', 'requirements.txt', 'static', 'templates', 'util.py']
radiographyml | 09:50:20 ['my_model1.h5']
radiographyml | 09:50:20 Traceback (most recent call last):
radiographyml | 09:50:20   File "app.py", line 53, in <module>
radiographyml | 09:50:20     model = load_model('/app/models/my_model1.h5')
radiographyml | 09:50:20   File "/usr/local/lib/python3.7/site-packages/tensorflow/python/keras/saving/save.py", line 186, in load_model
radiographyml | 09:50:20     loader_impl.parse_saved_model(filepath)
radiographyml | 09:50:20   File "/usr/local/lib/python3.7/site-packages/tensorflow/python/saved_model/loader_impl.py", line 113, in parse_saved_model
radiographyml | 09:50:20     constants.SAVED_MODEL_FILENAME_PB))
radiographyml | 09:50:20 OSError: SavedModel file does not exist at: /app/models/my_model1.h5/{saved_model.pbtxt|saved_model.pb}
```

I'm printing out the folder contents prior to the load_model error to prove that the model does exist. Anyone with experience deploying a custom model using this project to a cloud server?

**Comment by mtobeiyf** [comment]
Hi, I've used this repo to deploy my own toy project into the cloud. 
As I read the log, it seems like some issue with loading the model. `SavedModel file does not exist at: ./models/my_model.h5/{saved_model.pbtxt|saved_model.pb}`
I guess the path to your model may be wrong. Are you using a relative path for the model? Try switching to an absolute path for the application to find the model.

**Comment by reconstructive** [comment]
Hi there, thanks for the response!

Yes I've Googled this issue to death, and tried the absolute vs relative filepath approaches and nothing is working.

I feel like it may be a git lfs to github to digital ocean issue. My model is 275mb and so I needed to use git lfs to push it into my github repo. When attempting to deploy the app I printed the model file size and it shows it as 134 bytes which is clearly an issue. I'm currently engaging with the digital ocean support engineers so hopefully they can assist. 

I guess this is a cloud deployment issue so I'll close this one. Thanks again!
